docs/schema-loader.md

```diff
@@ -1,50 +1,83 @@
 # ScalarDB Schema Loader
 
-ScalarDB has its own data model and schema, that maps to the implementation specific data model and schema.
-Also, it stores internal metadata (e.g., transaction ID, record version, transaction status) for managing transaction logs and statuses when you use the Consensus Commit transaction manager.
-It is a little hard for application developers to manage the schema mapping and metadata for transactions, so we offer a tool called ScalarDB Schema Loader for creating schema without requiring much knowledge about those.
+ScalarDB has its own data model and schema that maps to the implementation-specific data model and schema. In addition, ScalarDB stores internal metadata, such as transaction IDs, record versions, and transaction statuses, to manage transaction logs and statuses when you use the Consensus Commit transaction manager.
 
-There are two ways to specify general CLI options in Schema Loader:
-  - Pass a ScalarDB configuration file and database/storage-specific options additionally.
-  - Pass the options without a ScalarDB configuration (Deprecated).
+Since managing the schema mapping and metadata for transactions can be difficult, you can use ScalarDB Schema Loader, which is a tool to create schemas that doesn't require you to need in-depth knowledge about schema mapping or metadata.
 
-Note that this tool supports only basic options to create/delete/repair/alter a table. If you want
-to use the advanced features of a database, please alter your tables with a database specific tool after creating them with this tool.
+You have two options to specify general CLI options in Schema Loader:
 
-# Usage
+- Pass the ScalarDB properties file and database-specific or storage-specific options.
+- Pass database-specific or storage-specific options without the ScalarDB properties file. (Deprecated)
```

Comment:
Should we keep the deprecated database-specific options in this doc on `master` and all version branches? Or should we remove the deprecated database-specific options from all or some versions of docs?